scanner/metadata/taglib.go

```diff
@@ -1,83 +1,13 @@
+//+build !cgo
```

Comment:
Small nitpick: I prefer to have this file named `taglib_nocgo.go` and keep the other one as `taglib.go`